cypress/features/regression/accessibility/accessibilityCommon.feature

```diff
@@ -31,47 +31,48 @@ Feature: Accessibility tests - Common list
       | flash              |
       | pages              |
 
-@accessibility
-Scenario Outline: Component <component> default story
-  When I open "<component>" component page "default" in no iframe
-  Then "<component>" component has no accessibility violations
-  Examples:
-    | component           |
-    | app wrapper         |
-    | carousel            |
-    | card                |
-    | configurable-items  |
-    | content             |
-    | detail              |
-    | draggableContext    |
-    | heading             |
-    | help                |
-    | i18ncomponent       |
-    | icon                |
-    | link                |
-    | loader              |
-    | menulist            |
-    | message             |
-    | mount-in-app        |
-    | multi-action-button |
-    | pill                |
-    | portrait            |
-    | preview             |
-    | profile             |
-    | row                 |
-    | settingsrow         |
-    | split-button        |
-    | table-ajax          |
-    | table               |
-    | tooltip             |
+  @accessibility
+  Scenario Outline: Component <component> default story
+    When I open "<component>" component page "default" in no iframe
+    Then "<component>" component has no accessibility violations
+    Examples:
+      | component           |
+      | button toggle       |
+      | carousel            |
+      | card                |
+      | configurable-items  |
+      | content             |
+      | draggableContext    |
+      | i18ncomponent       |
+      | icon                |
+      | link                |
+      | loader              |
+      | menulist            |
+      | message             |
+      | mount-in-app        |
+      | multi-action-button |
+      | pill                |
+      | pod                 |
+      | portrait            |
+      | preview             |
+      | profile             |
+      | row                 |
+      | showeditpod         |
+      | settingsrow         |
+      | split-button        |
+      | step-sequence-item  |
+      | step-sequence       |
+      | table-ajax          |
+      | table               |
+      | tooltip             |
 
-@accessibility
```

Comment:
those tests shouldn't be deleted... but only added the new yours one

Reply:
Fixed in PR #3324